File: GRASP/expert_creator/java/Produto.java
public class Produto{
    private String descricao;
    private float preco;
    private String upc;

    public Produto(String descricao, float preco, String upc) {
        this.descricao = descricao;
        this.preco = preco;
        this.upc = upc;
    }

    public float getPreco()   {
        return this.preco;
    }
}

File: GRASP/controller_polymorphism/produto.py
class Produto():
    def __init__(self, descricao, preco, upc):
        self._descricao = descricao
        self._preco = preco
        self._upc = upc

    def get_preco(self):
        return self._preco

    def get_descricao(self):
        return self._descricao